quilt/tools/build.py

```diff
@@ -116,7 +139,17 @@ def _build_node(build_dir, package, name, node, fmt, target='pandas', checks_con
                 print("Saving as binary dataframe...")
                 package.save_df(dataframe, name, rel_path, transform, target, fmt)
 
-def _file_to_spark_data_frame(ext, path, target, user_kwargs):
+def _remove_keywords(d):
+    """
+    copy the dict, filter_keywords
+
+    Parameters
+    ----------
+    d : dict
+    """
+    return { k:d[k] for k in d if k not in RESERVED }
```

Comment:
Not that it really matters here, but this is more Pythonic:

```{ k: v for k, v in iteritems(d) if k not in RESERVED }```

(May need to import `iteritems` from `six`.)